crates/jit-icache-coherence/src/libc.rs

```diff
@@ -60,6 +60,38 @@ pub(crate) fn pipeline_flush_mt() -> Result<()> {
         // In any other case we got an actual error, so lets propagate that up
         e => e?,
     }
+    #[cfg(all(target_arch = "riscv64", target_os = "linux"))]
+    if num_cpus::get() == 1 {
```

Comment:
`num_cpus::get()` does multiple syscalls and as such may be more expensive than the `SYS_RISCV_FLUSH_ICACHE_ALL` syscall it is meant to avoid.

Reply:
Ok,My main consideration is that detected at runtime make it is more easy to use. use this library need no configuration.Maybe I will add a feature `one-core`. Because of multi-core is the most case.